user/frawk/patches/cargo-config.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/.cargo/config b/.cargo/config
```

Comment:
you both patch this and self.rm it (just the self.rm is fine)

Reply:
hm, if you prefer it that way that's ok, in that case you can remove the self.rm since the patch is removing it

Reply:
it is actually not: The patch is clearing out the file, but it remains as an empty file afterwards, which still breaks.

After thinking about it some more: If they end up adding something to the cargo config that is required, it still compiling but still being broken subtly is probably very unlikely, so I'm fine with having the patch removed.

Reply:
i think this is a quirk of bsd patch, that it can't remove files then (since it was deleted in the patch itself) :)  

yeah, sounds good. thanks for checking!